packages/carbon-graphs/CHANGELOG.md

```diff
@@ -2,4 +2,7 @@
 
 ## Unreleased
 
+* Changed
+  * Updated reflow in Graph and Gantt constructs to update the eventlines.
+
```

Comment:
Nit: I would move this to below `Migrated to the terra-graphs mono-rep.` so that this is in chronological order.

Reply:
Updated [here](https://github.com/cerner/terra-graphs/pull/15/commits/e16b4acc74720734b965f76a847fe0d646e2d7d1).